lib/acm_bot/utils.ex

```diff
@@ -0,0 +1,10 @@
+defmodule AcmBot.Utils do
```

Comment:
Why `AcmBot.Utils`, why not `AcmBot.Board` and having `info/0` ?

Reply:
Same reason as the `List.fold`, I'm used to make `Project.Utils` modules with functions that don't have any module associated (format related functions, etc.), although I consider this a bad practice myself :joy: